lib/config/fileconf.go

```diff
@@ -1935,6 +1935,9 @@ type WindowsDesktopService struct {
 	// Hosts is a list of static Windows hosts connected to this service in
 	// gateway mode.
 	Hosts []string `yaml:"hosts,omitempty"`
+	// NonADHosts is a list of static Windows hosts connected to this service in
+	// gateway mode that are not part of Active Directory.
+	NonADHosts []string `yaml:"non_ad_hosts,omitempty"`
```

Comment:
```suggestion
	// NonADHosts is a list of standalone Windows hosts that are not
	// jointed to an Active Directory domain.
	NonADHosts []string `yaml:"non_ad_hosts,omitempty"`
```

Let's not mention "gateway mode" just yet, as that's technically something different.